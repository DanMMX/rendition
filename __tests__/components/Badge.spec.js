/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import Provider from '../../src/components/Provider'
import Badge from '../../src/components/Badge'

test('Badge renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Badge text='badge' />
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
