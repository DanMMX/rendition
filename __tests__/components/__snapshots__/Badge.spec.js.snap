// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge renders correctly 1`] = `
<div
  className="sc-bdVaJa jKopLr"
>
  <div
    className="sc-htoDjs gDCMNh sc-gzVnrw jIyoHV sc-bZQynM hnmAWT sc-ifAKCX fWjQQu sc-bwzfXH eMpePF"
  >
    badge
  </div>
</div>
`;
